pubsub/pubsubApiClient.go

```diff
@@ -48,32 +50,24 @@ func (ac *apiClient) SubscriptionForTopic(ctx context.Context, message pscontrac
 
 	projectID := message.GetProject()
 	subscriptionName := message.GetSubscription()
-
-	span.SetTag("project", projectID)
-	span.SetTag("subscription", subscriptionName)
-
-	if strings.HasSuffix(subscriptionName, ac.config.SubscriptionNameSuffix) {
-		return &manifest.EstafettePubSubEvent{
-			Project: projectID,
-			Topic:   strings.TrimSuffix(subscriptionName, ac.config.SubscriptionNameSuffix),
-		}, nil
-	}
-
-	subscription := ac.pubsubClient.SubscriptionInProject(subscriptionName, projectID)
-	if subscription == nil {
-		return nil, fmt.Errorf("Can't find subscription %v in project %v", subscriptionName, projectID)
+	topicID, err := message.GetTopic(topicAttributeName)
+	if err != nil {
+		if strings.HasSuffix(subscriptionName, ac.config.SubscriptionNameSuffix) {
+			topicID = strings.TrimSuffix(subscriptionName, ac.config.SubscriptionNameSuffix)
+		}
 	}
 
-	subscriptionConfig, err := subscription.Config(context.Background())
```

Comment:
I am not sure that it was working as expected.
Now case when topic is not in attributes and topic can't be evaluated from subscription name will result in error